package/batocera/core/batocera-configgen/scripts/tdp_hooks.sh

```diff
@@ -20,7 +20,15 @@ fi
 
 # set the final tdp value
 set_tdp() {
-    echo "Game ${2} requested setting AMD Mobile Processor TDP to ${1} Watts" >> $log
+    TDP_VALUE=$1
+    ROM_NAME=$2
+
+    if [ -z "${ROM_NAME}" ]; then
+        echo "Setting AMD Mobile Processor TDP to ${1} Watts" >> $log
```

Comment:
Why continue to use `${1}` and `${2}`? Shouldn't you use the variables you created for readability?